<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValidStylesDialog</class>
 <widget class="QDialog" name="ValidStylesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <widget class="QCheckBox" name="unknown">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="waypoint">
       <property name="text">
        <string>Waypoint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="airfieldGrass">
       <property name="text">
        <string>Airfield with grass surface runway</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="outlanding">
       <property name="text">
        <string>Outlanding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="airfieldGliding">
       <property name="text">
        <string>Gliding airfield</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="airfieldSolid">
       <property name="text">
        <string>Airfield with solid sutface</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mountainPass">
       <property name="text">
        <string>Mountain Pass</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mountainTop">
       <property name="text">
        <string>Mountain Top</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="transmitterMast">
       <property name="text">
        <string>Transmitter Mast</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="vor">
       <property name="text">
        <string>VOR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ndb">
       <property name="text">
        <string>NDB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="coolingTower">
       <property name="text">
        <string>Cooling Tower</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="dam">
       <property name="text">
        <string>Dam</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="tunnel">
       <property name="text">
        <string>Tunnel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bridge">
       <property name="text">
        <string>Bridge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="powerPlant">
       <property name="text">
        <string>Power Plant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="castle">
       <property name="text">
        <string>Castle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="intersection">
       <property name="text">
        <string>Intersection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ValidStylesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ValidStylesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
